status.html

```diff
@@ -1,124 +1,69 @@
 ---
-title: Component Status
+title: Pattern Status
 layout: docs
+category: Status
+usage: >
+  Refer to this page for an alphabetical list of all patterns in Helix, including design and code statuses.
 ---
 
-<article>
-  <p>Listing of components in the language and their current status. Lorem Ipsum.</p>
-<!--
-  <h2>Option 1</h2>
-    <table class="striped">
+<div class="hxRow">
+<div class="hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8">
+
+<table class="hxTable hxTable--condensed hxTable--bordered" id="design-status-table">
     <thead>
       <tr>
-        <th style="width: 33%">Component</th>
-        <th style="width: 33%">Version Added</th>
-        <th style="width: 33%">Status</th>
+        <th>Helix Pattern</th>
+        <th class="align-center">Design Doc</th>
+        <th class="align-center">CSS</th>
+        <th class="align-center">JS</th>
       </tr>
     </thead>
     <tbody>
-      {%- for page in site.pages -%}
-          {%- if page.category -%}
-              <tr>
-                <td><a href="{{site.baseurl}}{{page.url}}.html">{{page.title}}</a></td>
-                <td>1.0</td>
-                <td>
-                {%- if page.status -%}
-                  {%- if page.status == "pending" -%}
-                    <span class="component prototype">In Progress</span>
-                  {%- elsif page.status == "complete" -%}
-                    <span class="component dev-ready">Complete</span>
-                  {%- elsif page.status == "deprecated" -%}
-                    <span class="component deprecated">Deprectated</span>
-                  {%- endif -%}
-                {%- else -%}
-                  <span class="component prototype">In Progress</span>
-                {%- endif -%}
-                </td>
-              </tr>
-          {%- endif -%}
-      {%- endfor -%}
+      {% assign component_list = "" | split: ','%}
```

Comment:
minor nit: If you use a hyphen inside the percent signs (`{%- ... -%}`), the resulting pages don't have a ton of whitespace in the source code.

Reply:
oh man!  I did not know that!  Thanks for the Pro-Tip!